project/Dependencies.scala

```diff
@@ -6,7 +6,8 @@ object Dependencies {
   )
 
   lazy val testDependencies = Seq(
-    "org.scalatest" %% "scalatest" % "3.2.12",
-    "org.scalanlp" %% "breeze" % "2.0"
+    "org.scalatest"           %% "scalatest"                  % "3.2.12",
+    "org.scalanlp"            %% "breeze"                     % "2.0",
+    "org.scala-lang.modules"  %% "scala-parallel-collections" % "1.0.0"
```

Comment:
Is this still necessary on newer scala versions?

Reply:
Yes, it's necessary for 2.13. The [first build](https://app.travis-ci.com/github/CitrineInformatics/sprandom/builds/256136928) (before this dependency was added) failed with `object parallel is not a member of package collection`